### DISCUSSION TITLE
Enable JMX in Strimzi kafka

### DISCUSSION THREAD

**Discussion by zarafree** [strimzi/strimzi-kafka-operator#6318] [Q&A]
We are using 0.21 strimzi deployed in Kubernetes environment and we want to monitor some of kafka metrics through third party monitoring tool like datadog or similar(Not Prometheus). We have enabled jmxoption {} setting in kafka resource definition but still unable to get kafka metrics through service endpoint over 9999 port. We are getting error "Empty reply from server" while accessing through curl.  Can you please guide us how to enable JMX settings in Strimzi Kafka?

**Comment by michael-barker** [comment]
This should be resolved by https://github.com/strimzi/strimzi-kafka-operator/issues/7535.

**Comment by andreyolv** [comment]
I have the same problem, but using version 0.32
